package.json

```diff
@@ -8,7 +8,7 @@
   },
   "scripts": {
     "lint": "./node_modules/.bin/standard --verbose src/**",
-    "test": "rm -rf ./tmp ; jest --forceExit --detectOpenHandles --coverage --runInBand --testURL=\"http://localhost\"",
```

Comment:
why are you removing `--forceExit`? running test fails because of a timeout, at least at the CI. We can create a special method for the CI though

Reply:
Yeah, just noticed this. I removed it to get the console output locally. Readded now.